### DISCUSSION TITLE
ReazonSpeech v1.1.0をリリースしました！

### DISCUSSION THREAD

**Discussion by fujimotos** [reazon-research/ReazonSpeech#13] [Announcements]
ReazonSpeech v1.1.0を2023年4月4日にリリースしました。

* リリースノート
  * https://github.com/reazon-research/ReazonSpeech/releases/tag/v1.1.0
* ブログ記事
  * https://research.reazon.jp/blog/2023-04-04-ReazonSpeech.html

新しいインターフェイスの使い方に関する質問や、聞きたいポイントがあれば
このスレッドでフィードバックください！

**Comment by tomohiro5560** [comment]
質問です。
Reason Speachを使って日本語の動画の文字起こしを行いました。
私の行いたい点としては、fillerと呼ばれる「えー」や「あー」や「えっと」のような言葉や言いよどみも含めて動画で言った言葉そのまま文字起こしを行いたいです。
Reason Speachを使ってbeam sizeを変更したり、inputする動画の速度を変えたり実験を試したのですが、なかなか精度が上がらないため開発者に質問をした次第です。
何かいい解決方法やよいoptionなどあればご教示いただきたいです。
宜しくお願いいたします。

**Reply by fujimotos** [reply]
> 私の行いたい点としては、fillerと呼ばれる「えー」や「あー」や「えっと」のような言葉や言いよどみも含めて動画で言った言葉そのまま文字起こしを行いたいです。

これは放送字幕データ特有の性質だと思います。

もともと放送字幕では「えっと」「まあ」といった場つなぎの挿入語は除いて付与されている事が多く、
この学習データをもとにモデルを訓練すると、自然とフィラー抜きのテキストが推論されます。

> 何かいい解決方法やよいoptionなどあればご教示いただきたいです

もしフィラーを含めて推論したい場合は、フィラーを含むコーパスを準備して、
モデルを学習することが必要になると思います。
